src/services/importTracker.ts

```diff
@@ -130,6 +133,19 @@ namespace ts.FindAllReferences {
                                 directImports.push(direct);
                             }
                             break;
+
+                        case SyntaxKind.ImportType:
+                            if (direct.qualifier) {
+                                // `import("foo").x` named import
+                                directImports.push(direct);
+                            }
+                            else {
+                                // TODO: GH#23879
+                            }
+                            break;
+
+                        default:
+                            Debug.assertNever(direct);

```

Comment:
I'd include a helpful message here as the second parameter to `assertNever`, just in case (if the types drift out of sync with reality because of an invalid cast, for example).